lib/pymedphys/cli/experimental/pinnacle.py

```diff
@@ -59,7 +59,7 @@ def export_pinnacle(pinnacle_subparsers):
     parser.add_argument(
         "--mrn",
         action="store_true",
-        help=("Ammend the Patient ID/HRN to the end of the name of the export folder"),
+        help=("Ammend the Patient ID/MRN to the end of the name of the export folder"),
```

Comment:
Amend

Reply:
Thank you. Replaced with append as it is adding to the end of a string